### ISSUE TITLE
Validation loss log 제출 관련 문의

### ISSUE THREAD

**Issue by sebinkim** [LISTatSNU/FastMRI_challenge#275]
안녕하세요, 세빈민균 팀입니다.

현재 대회 기간이 얼마 남지 않아, 저희 팀에서는 여러 대의 GPU를 이용해 나눠서 학습을 진행한 뒤 하나의 모델로 합치는 전략을 시도하고 있습니다.

이렇게 될 경우 작은 모델이 따로 학습을 진행하기 때문에 validation loss가 여러 개 생기게 되는데요,

모델 weight의 경우 하나의 큰 파일로 합쳐서 제출할 수 있지만 validation loss는 그렇게 하기 어렵습니다.

따라서 각각의 모델을 학습시키는 코드와 이에 대한 validation loss를 따로 제출하는 것이 가능한지에 대해 문의드립니다.

감사합니다.

**Comment by jslee525** [comment]
안녕하세요, 2024 SNU FastMRI Challenge 조교 이준성입니다.

Model Ensemble에 대한 제출 양식을 여쭤본 것으로 이해됩니다,
기존에 안내드렸던 것처럼 폴더 구조를 맞추어야 하지만, 모델이 여러 개인 경우에는 안내드렸던 폴더 구조에서 최대한 벗어나지 않게 제출해주신 후에 readme.txt에 적어주시면 됩니다.

예를 들어, val_loss_log.npy를 제출하는 위치에 val_loss_log_[model A].npy, val_loss_log_[model B].npy ... 등으로 제출해주시면 됩니다. 다만, 저희가 실제로 검증을 할 때 제출해주셨던 validation loss가 어떠한 training code에 의해서 만들어진 것인지 적어주셔야 합니다. (여러 모델을 검증해야 하기 때문에 헷갈릴 수 있습니다.) 필수적인 것은 아니지만 가능하시다면, training code를 돌릴 경우 자동으로 npy 파일이 생성될 텐데 그 파일 이름에 모델 이름을 넣어주세요. (ex. Unet training을 돌렸을 때 val_loss_log_unet.npy 생성 / Varnet training을 돌렸을 때 val_loss_log_varnet.npy 생성)

감사합니다.
이준성 드림
